code/WorkInProgress/Ported/policetape.dm

```diff
@@ -144,7 +144,7 @@
 		for(var/atom/A in T) //Check to see if there's anything solid on the tape's turf (it's possible to build on it)
 			if(A.density)
 				return
-		M:loc = T
+		M:forceMove(T)
```

Comment:
@DrCelt expanding on what @ComicIronic said, for ```/proc```s any ```X as Y``` statement in the arguments does literally nothing, it's a construct designed for ```/verb```s, it compiles in procs because byond is stupid at times.